app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java

```diff
@@ -46,7 +46,7 @@
 
     private static final int LOCATION_REQUEST = 1;
     private static final String MAP_LAST_USED_PREFERENCE = "mapLastUsed";
-
+    private boolean LOCATION_CHANGED=false;
```

Comment:
please refer to google style guide for private variable naming conventions. :)

Reply:
Sorry :p . I know that its an elementary mistake, I'll correct it.